src/register-shared-deps.ts

```diff
@@ -0,0 +1,9 @@
+import * as singleSpa from "single-spa";
```

Comment:
This file is worth looking over. It is how we're loading the shared dependencies that all applications need in order to work.